.gitattributes

```diff
@@ -0,0 +1,5 @@
+/.github export-ignore
+/.gitignore export-ignore
+/CONTRIBUTING.md export-ignore
+/.markdownlint.json export-ignore
+/.editorconfig export-ignore
```

Comment:
Why the need to disable these for archives?

Reply:
It is not "needed" but a nice to have
People do not care about dev files when they download your repo `git archive`: https://github.com/rust-unofficial/awesome-rust/archive/refs/heads/master.zip